electron/electron.ts

```diff
@@ -23,28 +23,30 @@ log.info("App starting...");
 if (app.isPackaged) {
   const dev = isDev();
 
-  Sentry.init({
-    environment: dev ? "development" : "production",
-    dsn: "https://5ff29bb5b3b64cd4bd4f4960ef1db2e3@o4504750197899264.ingest.sentry.io/4504750206746624",
-    debug: true,
-    release: app.getVersion(),
-    transportOptions: {
-      maxQueueAgeDays: 30,
-      maxQueueCount: 30,
-      beforeSend: async () => {
-        const ol = await net.isOnline();
-        return ol ? "send" : "queue";
+  if (!dev) {
+    Sentry.init({
+      environment: dev ? "development" : "production",
```

Comment:
This line isn't really a trinary inside the if.

Reply:
good point.